source/administration/config-settings.rst

```diff
@@ -3732,6 +3740,17 @@ Changes made when hardened mode is enabled:
 | This feature’s ``config.json`` setting is ``"ExperimentalEnableHardenedMode": false`` with options ``true`` and ``false`` for above settings respectively.          |
 +---------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Limit Access to Config Settings Prior to Login
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Enable this setting to limit the number of config settings sent to users prior to login.
+
+Supported for Mattermost server v5.1.0 and later, and Mattermost Mobile apps v1.9.0 and later.
+
++-------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature’s ``config.json`` setting is ``"ExperimentalLimitClientConfig": "false"`` with options ``true`` and ``false`` for above settings respectively.       |
```

Comment:
Just curious why this is experimental and set to false by default?

Reply:
@esethna Because this is a breaking change for older apps, this setting is disabled by default with an experimental setting to turn it on for testing. Once we get to a point where we drop support for the 1.9 mobile apps, we can remove the setting and enable it for everyone.